rageshake.sample.yaml

```
@@ -20,3 +20,17 @@ github_project_mappings:
 # a Slack personal webhook URL (https://api.slack.com/incoming-webhooks), which
 # will be used to post a notification on Slack for each report.
 slack_webhook_url: https://hooks.slack.com/services/TTTTTTT/XXXXXXXXXX/YYYYYYYYYYY
+
+# notification can also be pushed by email.
+# this param controls the target emails
+email_addresses:
+- support@matrix.org
```

I think this is clearer:

```suggestion
  - support@matrix.org
```